scripts/post_deploy.py

```diff
@@ -26,11 +27,16 @@
 
 tdr = TDRClient.with_service_account_credentials()
 public_tdr = TDRClient.with_public_service_account_credentials()
+unregistered_tdr = TDRClient.with_unregistered_service_account_credentials()
 
 
 def register_with_sam():
     tdr.register_with_sam()
     public_tdr.register_with_sam()
+    unregistered_email = unregistered_tdr.credentials.service_account_email
+    require(not tdr.user_is_registered(unregistered_email),
+            'The "unregistered" service account has been registered',
+            unregistered_email)
```

Comment:
I think we should minimize the changes to production code that are required to provision the third service account. We only need that SA for testing. Instead of validating this here, could the IT enforce this?

Also, one SAM user can check if another SAM user is registered? This seems odd.

Either way, this should just use `GET /register/user/v1` as that is much simpler.